frb_dart/lib/src/isolate/stub.dart

```diff
@@ -0,0 +1,4 @@
+import 'dart:isolate';
+export 'dart:isolate';
+
+ReceivePort broadcastPort(String portName) => throw UnimplementedError();
```

Comment:
nit: Seems possible to remove the stub.dart - we always conditionally import one of io or web, which always has real impl?

Reply:
The stub exists because we are defining other items on top of `dart:isolate` which cannot be intercepted, but we still need them to be visible for editing purposes. Also, conditional imports are somewhat brittle on Web as explained in dart-lang/sdk#46918.

Reply:
I see. Btw that issue is closed - so if the problem still exists maybe we need to create a new issue, or write down a comment there, o/w Dart people may consider it solved.

Reply:
AFAIK the problem was never "solved" and has been put on the backlog. I'll keep the stub module since it is generally useful, not unless the Dart team decides to revamp web modules in the (hopefully not so far) future.